Package.swift

```diff
@@ -1,9 +1,9 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.9
 import PackageDescription
 
 let package = Package(
     name: "Ashton",
-    platforms: [.iOS("13.4"), .macOS(.v10_15)],
+    platforms: [.iOS("13.4"), .macOS(.v10_15), .visionOS(.v1)],
```

Comment:
Trying to use `.visionOS(…)` when the `swift-tools-version` is lower than `5.9` doesn’t seem to work 😞